dot/state/storage_notify.go

```diff
@@ -74,3 +77,16 @@ func (s *StorageState) notifyChanged(change *KeyValue) {
 		}(ch)
 	}
 }
+
+func (s *StorageState)Subscribe(prefixes []byte) {
+	fmt.Printf("SUBSCRIBE CALLED db %v pre %x\n", s.db, prefixes)
```

Comment:
Can we replace this with logger debug/trace?

Reply:
Yes, I will (this was from a false commit from another branch)